MaxText/layers.py

```diff
@@ -1071,16 +1093,17 @@ def __call__(self,
         dtype=cfg.dtype,
         name='mlp',
         config=cfg,
-    )(lnx, deterministic=deterministic)
+    )(hidden_states, deterministic=deterministic)
     mlp_lnx = nn.with_logical_constraint(mlp_lnx, ('activation_batch', 'activation_length', 'activation_embed'))
 
-    next_layer_addition = mlp_lnx + attention_lnx
-
-    next_layer_addition_dropped_out = nn.Dropout(
-        rate=cfg.dropout_rate, broadcast_dims=(-2,))(
-            next_layer_addition, deterministic=deterministic)
+    layer_output = mlp_lnx + residual
+    
+    #No Dropout in Llama
```

Comment:
Can't we just set dropout rate to zero? (Sorry I'm copy and pasting this one haha)

Reply:
+1

Reply:
Done